theano/configdefaults.py

```diff
@@ -1238,21 +1252,20 @@ def default_blas_ldflags():
                                           "mk2_rt"]])
 
         # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
+        if "Continuum Analytics" in sys.version:
```

Comment:
I suppose conda-forge still use the python from anaconda and just add packages. Do you have other information? If not, I'll keep the old condition and add a comment that this is needed for miniconda and anaconda.

Another idea, why not change the condition to "if os.path.exists(...)

     mingw_w64_gcc = os.path.join(os.path.dirname(sys.executable), "Library", "mingw-w64", "bin", "g++")
     if os.path.exists(mingw_w64_gcc):

that way, if the path exist, we will try it. It will help prevent to start a new process when not needed.